migration-guides/v.10.0.0.md

```diff
@@ -0,0 +1,53 @@
+# v10.0.0 Migration Guide
+
+This version contains the following breaking changes:
+
+1. The `DropdownCheckboxGroup` component is now `CheckboxGroup` with prop `dropdown`=`true`
+2. The `CheckboxGroup` and `RadioGroup` legends now rely on `visually-hidden` class styles to hide the label from the view
+
```

Comment:
I think we're missing a big breaking change here – `fieldset` no longer wraps each input! This will affect base styles in `_forms.scss`, as well as any **custom** rules that rely on this structure

Reply:
I also added an additional call out about `LabeledField` itself (bullet point 2) since that is an exported component that can be used on its own.. thoughts?